modules/veteran/app/controllers/veteran/v0/flag_accredited_representatives_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Veteran
+  module V0
+    class FlagAccreditedRepresentativesController < ApplicationController
+      service_tag 'lighthouse-veteran'
+      skip_before_action :verify_authenticity_token
+      skip_before_action :authenticate
+      before_action :feature_enabled
+
+      def create
+        FlaggedVeteranRepresentativeContactData.transaction do
+          flags = params[:flags].map do |flag_data|
+            FlaggedVeteranRepresentativeContactData.new(
+              flag_data.permit(:flag_type, :flagged_value).merge(
+                ip_address: request.remote_ip,
+                representative_id: params[:representative_id]
+              )
+            )
+          end
+
+          if flags.all?(&:valid?)
+            flags.each(&:save!)
+            render json: flags, status: :created
```

Comment:
I think the render should be outside of the transaction block

Reply:
TY. Fixed.